Makefile

```diff
@@ -49,11 +50,67 @@ build:
 		-ldflags $(GO_LDFLAGS) \
 		main.go
 
+### clean-image:		clean apisix-ingress-controller image
+.PHONY: clean-image
+clean-image: ## Removes local image
+	echo "removing old image $(REGISTRY)/apache/apisix-ingress-controller:$(IMAGE_TAG)"
+	docker rmi -f $(REGISTRY)/apache/apisix-ingress-controller:$(IMAGE_TAG) || true
+
 ### build-image:          Build apisix-ingress-controller image
 .PHONY: build-image
 build-image:
 	docker build -t apache/apisix-ingress-controller:$(IMAGE_TAG) --build-arg ENABLE_PROXY=true .
 
+### pack-ingress-image:  Build and push Ingress image used in e2e test suites to kind or custom registry.
+.PHONY: pack-ingress-image
+pack-ingress-image:
+	docker build -t apache/apisix-ingress-controller:$(IMAGE_TAG) --build-arg ENABLE_PROXY=$(ENABLE_PROXY) .
+	docker tag apache/apisix-ingress-controller:$(IMAGE_TAG) $(REGISTRY)/apache/apisix-ingress-controller:$(IMAGE_TAG)
+	docker push $(REGISTRY)/apache/apisix-ingress-controller:$(IMAGE_TAG)
+
+### pack-images: Build and push images used in e2e test suites to kind or custom registry.
+.PHONY: pack-images
+pack-images: build-images push-images
+
+### build-image:          Build apisix-ingress-controller image
+.PHONY: build-images
+build-images:
+ifeq ($(E2E_SKIP_BUILD), 0)
+	docker pull apache/apisix:2.13.1-alpine
+	docker tag apache/apisix:2.13.1-alpine $(REGISTRY)/apache/apisix:$(IMAGE_TAG)
+
+	docker pull bitnami/etcd:3.4.14-debian-10-r0
+	docker tag bitnami/etcd:3.4.14-debian-10-r0 $(REGISTRY)/bitnami/etcd:$(IMAGE_TAG)
+
+	docker pull kennethreitz/httpbin
+	docker tag kennethreitz/httpbin $(REGISTRY)/kennethreitz/httpbin:$(IMAGE_TAG)
+
+	docker build -t test-backend:$(IMAGE_TAG) --build-arg ENABLE_PROXY=$(ENABLE_PROXY) ./test/e2e/testbackend
+	docker tag test-backend:$(IMAGE_TAG) $(REGISTRY)/test-backend:$(IMAGE_TAG)
+
+	docker build -t apache/apisix-ingress-controller:$(IMAGE_TAG) --build-arg ENABLE_PROXY=$(ENABLE_PROXY) .
+	docker tag apache/apisix-ingress-controller:$(IMAGE_TAG) $(REGISTRY)/apache/apisix-ingress-controller:$(IMAGE_TAG)
+
+	docker pull jmalloc/echo-server:latest
+	docker tag  jmalloc/echo-server:latest $(REGISTRY)/jmalloc/echo-server:$(IMAGE_TAG)
+
+	docker pull busybox:1.28
+	docker tag  busybox:1.28 $(REGISTRY)/busybox:$(IMAGE_TAG)
```

Comment:
I do not recommend unifying the tags of these images into dev.
If the task fails, we need to check the code to know which image is used.

Can you explain your benefit of unifying them?

Reply:
The advantage of this is that because we use a local repository, we don't need to care about these images tags in the code. When upgrading, we only need to change the `Makefile` file. 
At present, `localhost:5000` is used to identify the local warehouse in our code. It's not difficult to track this problem.

Reply:
make sense, let's move forward.